keyboards/yoichiro/lunakey_macro/keymaps/default/rules.mk

```diff
@@ -0,0 +1,3 @@
+AUDIO_ENABLE = yes
+RGBLIGHT_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
These shouldn't be enabled at the keymap level, for the default keymap.  Instead, they should be at the keyboard level.

Reply:
@drashna Thanks, I remove the flags from the default keymap definition.